src/main/java/roomescape/exception/GlobalExceptionHandler.java

```diff
@@ -6,24 +6,21 @@
 import org.springframework.http.ResponseEntity;
 import org.springframework.web.bind.annotation.ExceptionHandler;
 import org.springframework.web.bind.annotation.RestControllerAdvice;
+import roomescape.dto.ErrorResponse;
 
 @RestControllerAdvice
 public class GlobalExceptionHandler {
 
     @ExceptionHandler(InvalidException.class)
-    public ResponseEntity<Map<String, String>> invalidException(InvalidException ex) {
-        Map<String, String> response = new HashMap<>();
-        response.put("error", "Bad Request");
-        response.put("message", ex.getMessage());
-        return ResponseEntity.status(HttpStatus.BAD_REQUEST).body(response);
+    public ResponseEntity<ErrorResponse> invalidException(InvalidException ex) {
+        ErrorResponse errorResponse = new ErrorResponse("Bad Request", ex.getMessage());
+        return ResponseEntity.status(HttpStatus.BAD_REQUEST).body(errorResponse);
     }
 
     @ExceptionHandler(NotFoundReservationException.class)
-    public ResponseEntity<Map<String, String>> notFoundReservation(NotFoundReservationException ex) {
-        Map<String, String> response = new HashMap<>();
-        response.put("error", "Bad Request");
-        response.put("message", ex.getMessage());
-        return ResponseEntity.status(HttpStatus.BAD_REQUEST).body(response);
+    public ResponseEntity<ErrorResponse> notFoundReservation(NotFoundReservationException ex) {
+        ErrorResponse errorResponse = new ErrorResponse("Bad Request", ex.getMessage());
+        return ResponseEntity.status(HttpStatus.BAD_REQUEST).body(errorResponse);
     }
```

Comment:
```suggestion
    public ResponseEntity<ErrorResponse> notFoundReservation(NotFoundReservationException ex) {
        ErrorResponse errorResponse = new ErrorResponse("Bad Request", ex.getMessage());
        return ResponseEntity.status(HttpStatus.NOT_FOUND).body(errorResponse);
    }
```

404를 의도하고 작성하신 코드인가요?! (까먹으신 것 같아서...)

Reply:
앗 네 맞습니다 ErrorResponse에 "Bad Request"라고 적혀있는데, 새로 예외처리를 작성했지만 테스트 코드 통과 과정에서 애를 먹어서 논리가 맞지 않는 부분이 있네요.. 이 부분은 좀 더 확인해서 수정하도록 하겠습니다